### ISSUE TITLE
Release to ROS2 Rolling / Galactic

### ISSUE THREAD

**Issue by SteveMacenski** [ros-perception/perception_open3d#18]
Hi,

The deadline for projects to get into the first cut of Galactic is today! You don't have to do it right now, but it would be good to get this released in the next week or two so that it gets in by the first sync.

For rolling, you should create a new `ros2` branch that all ROS2 PRs are against (the "master" for ros2) and then cut distributions off of it. That way if you update `rolling`, it'll get pulled into the next distribution automatically (assuming no new changes were made requiring a new release)

**Comment by nkhedekar** [comment]
Yes, agreed. It would be good to release this, but I guess we have to wait till open3d is released? https://github.com/intel-isl/Open3D/issues/2110

**Comment by SteveMacenski** [comment]
Add Humble to that list :-)

**Comment by SteveMacenski** [comment]
Release made for Humble / ROS 2 but Galactic is not going to be addressed since it EOLs soon and 20.04 does not have open3D binaries.
